### PR TITLE
Add seed

### PR DESCRIPTION
**Pull request by gwenwhite**
I am not sure if I added the random.seed function to the proper area or not. So WIP.

### CODE REVIEW

File: MCMC/simulation.py [hunk 1]
```diff
@@ -20,6 +20,7 @@ def __init__(
             r_cut=2.5,
             energy_write_freq=100,
             trajectory_write_freq=10000,
+            seed = 20
```
Comment (chrisjonesBSU) [inline MCMC/simulation.py:23]: We'll need a comma here after the `20`. Also, for the sake of consistent formatting, remove the spaces on each side of the `=`

**Review by chrisjonesBSU — CHANGES_REQUESTED** [review]

**Comment by chrisjonesBSU** [comment]
Thanks for starting this, I left a comment with one change.  Also, can you add a `seed` parameter to the `init.py` file in `MCMC-flow` and make sure it is being passed into the `Simulation` class in `project.py`?  Let me know if you have any questions about this.

File: MCMC/simulation.py (hunk 1 above)
Comment (gwenwhite) [reply MCMC/simulation.py:23]: sounds good, thanks

**Comment by gwenwhite** [comment]
should I also add seed to sim.run in project.py?

**Comment by chrisjonesBSU** [comment]
I don't think we'll need to pass it to `run()` since there is no parameter in `run()` for seed.

**Comment by chrisjonesBSU** [comment]
I'm getting an error when trying to initialize a simulation:

```
File ~/cme/projects/advanced-sampling/Advanced-Sampling/MCMC/simulation.py:57, in Simulation.__init__(self, n_density, n_particles, r, r_cut, energy_write_freq, trajectory_write_freq, seed, energy_func, hard_sphere, restart, **kwargs)
     54 self.temperatures = []
     55 self._tps = []
---> 57 random.seed(seed=seed)

TypeError: Random.seed() got an unexpected keyword argument 'seed'
```

Check out the docs for `random.seed`

https://docs.python.org/3/library/random.html

File: MCMC/simulation.py [hunk 2]
```diff
@@ -51,6 +53,8 @@ def __init__(
         self.energies = [copy.deepcopy(self.energy)]
         self.temperatures = []
         self._tps = []
+        
+        random.seed(seed=seed)
```
Comment (chrisjonesBSU) [inline MCMC/simulation.py:57]: Getting an error on this line.

**Review by chrisjonesBSU — CHANGES_REQUESTED** [review]

File: MCMC/simulation.py (hunk 2 above)
Comment (gwenwhite) [reply MCMC/simulation.py]: Thanks for pointing that out. I changed `seed=seed` to just `seed` and the error went away.

**Review by chrisjonesBSU — APPROVED** [review]
LGTM, thanks!

**Pull request merged**